packages/documentation/src/pages/getting-started/common-tasks/test.mdx

```diff
@@ -1,3 +1,8 @@
+---
+title: Run tests
+tags: unit test, lint, end to end test, accessibility, 508
```

Comment:
we can add whatever we want to the front matter for indexing. I think it would be nice to index the section titles too but I didn't see them easily available in the nodes.